admin/google_search_console/class-gsc-issues.php

```diff
@@ -118,11 +121,17 @@ private function issue_compare( &$crawl_issues, $issue ) {
 	 * @return WPSEO_GSC_Issue
 	 */
 	private function create_issue( $issue ) {
+		// Ignore coding standards for object properties.
+		// @codingStandardsIgnoreStart
+		$response_code = $issue->responseCode;
+		$page_url = $issue->pageUrl;
+		// @codingStandardsIgnoreEnd
```

Comment:
Those properties should pass as long as they are on the whitelist as added in https://github.com/Yoast/wordpress-seo/pull/6875/commits/14f775fc7a7342dbf14e42a93870bf457b679d36